### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by ali-n4i**

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -139,3 +129,11 @@ https://5a3e964ed0b51.streamlock.net/w/azadah/chunklist_w1398305128.m3u8
 https://ir3.live21.ir/hls2/valiasr.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/7uEI6ck.png" group-title="News",VOA Persian
 https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/JyvW6Dn.png" group-title="News",BBC Persian
+https://www.youtube.com/watch?v=FCKruF2LZNs
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/oSSoK87.jpg" group-title="Entertainment",MBC Persia
+https://www.youtube.com/watch?v=Faqn9Anz0aw
```
Comment (freearhey) [inline channels/ir.m3u]: Sorry, but we don't add YouTube streams to the playlist.

File: channels/ir.m3u [hunk 2]
```diff
@@ -139,3 +129,11 @@ https://5a3e964ed0b51.streamlock.net/w/azadah/chunklist_w1398305128.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/Icob3E3.jpg" group-title="Entertainment",4U TV
+http://live.4ulive.ir/
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/l0RcbG4.jpg" group-title="Entertainment",Gala1
+http://184.104.200.39:7777/Gala1.m3u8?user=sglsbnf2kxaknar2cwx22rtj305v&session=3646036f88acb17d89327ba2231fc22ff0791720f2c5f8a7c5ea279e65654b81e58202f9653afd0d58fb9ce2805f23ae4378f293d12ea616&amp
```
Comment (freearhey) [inline channels/ir.m3u]: This link doesn't work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please make changes to the pull request so we can approve it.

**Comment by nazari-dev** [comment]
@freearhey Please check it out again.

**Review by freearhey — APPROVED** [review]

**Pull request merged**